doc/tutorials/increase-uos-disk-size.rst

```diff
@@ -0,0 +1,51 @@
+.. _Increase UOS disk size:
+
+Increasing the User OS disk size
+################################
+
+This document builds on the :ref:`getting_started` and assumes you already have
+a system with ACRN installed and running correctly. The size of the pre-built
+Clear Linux User OS (UOS) virtual disk is typically of 8GB. This may not be
+sufficient and this guide explains in a few steps how to increase the size of
+that virtual disk.
+
+This document is largely inspired from Clear Linux's `Increase virtual disk size
+of a Clear Linux* OS image <https://clearlinux.org/documentation/clear-linux/
+guides/maintenance/increase-virtual-disk-size>`_ tutorial. The process can be
+broken down in three steps:
+
+1. Increase the virtual disk size
+#. Resize the ``rootfs`` partition
+#. Resize the filesystem
+
+Increase the virtual disk size
+******************************
+
+.. note::
+
+   This step **must** be performed while the User OS (UOS) is powered off.
```

Comment:
How do you "power off" the UOS?

Reply:
I use `poweroff` in the UOS shell. I think it's also possible to use `acrnctl` in the SOS but I've never played with that.

Reply:
I was thinking you'd add this to the documentation, but maybe it's obvious to the trained user :)